File: python/tutorial/03-repeatquestion.py
# --------------------------------------------
# Repeating the question
# See http://www.tropo.com for more info
# --------------------------------------------

answer()

result = ask("Hi. For sales, press 1. For support, press 2.",
{'choices':"1,2", 'repeat':3})

if (result.name == 'choice'):
  if (result.value == "1"): say("Sales is not available right now")
  if (result.value == "2"): say("Support is currently on the other line.")

hangup()

File: ruby/tutorial/core/03-repeatquestion.rb
# -----------
# repeating the question
# -----------


answer

options = { :choices => '1, 2', 
            :repeat  => 3 }
            
result = ask 'For sales, press 1. For support, press 2.', options

if result.name == 'choice'
  case result.value
  when '1'
    say 'sales is not available right now.'
  when '2'
    say 'support is currently on the other line.'
  end
end

hangup